src/nxdoc/web-ui/web-ui-upgrade-notes/web-ui-upgrade-notes-lts-2023.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Upgrading Nuxeo Web UI from LTS 2021 to LTS 2023
+description: Upgrade notes from Nuxeo Web UI LTS 2021 to LTS 2023
+review:
+    comment: ''
+    date: '2023-03-28'
+    status: ok
+toc: true
+labels:
+tree_item_index: 998
```

Comment:
Can't check this one now (can't run a build of LTS 2023 documentation due to technical issues) but this probably needs to be changed. Was 550 for LTS 2021.